### ISSUE TITLE
How to render initial or RESET/REPLACE scenes with animation?

### ISSUE THREAD

**Issue by holden-caulfield** [aksonov/react-native-router-flux#1709]
### Version
Tell us which versions you are using: 

- react-native-router-flux v3.37.0
- react-native v0.38

TL; DR;
This is more a question than a bug report, I would like to present my initial screen (and other screens I navigate to with a RESET type) with a 'fade' animation, however it looks like the animations are only executed when we do a regular push of the stack. Is there a way to do that?
TL; DR;

A little bit more detail:
I have a splash screen with a loading spinner, defined like this:

`<Scene initial key='splash' component={Splash} hideNavBar animation='fade' duration=1000 />`

Than then navigates to my home scene by issuing:
`Actions.home({type: ActionConst.RESET, animation: 'fade', duration:1000})`

The desired effect is that both the splash screen and the home screen appear with a fade in transition. However in both cases the animation is not performed. 

For the home screen case, I've also tried using `ActionConst.REPLACE` with no luck. Unfortunately I cannot just push the home screen because if I do so the nav bar will offer to go back to the splash screen which is not ok.

Let me know if there is anything I could do differently to achieve the intended behavior

**Comment by nouveller** [comment]
I've been fighting around the same issue. I believe `ActionConst.POP_AND_REPLACE` does the inverse of this and we need `PUSH_AND_REPLACE`, so the animations can occur, and then the stack is reset.

I've not found a neat way to do this other than remove the animations for now.
